### ISSUE TITLE
Complex layer (All US Zipcodes) disappears on zoom out

### ISSUE THREAD

**Issue by gofishwbeeler** [mapbox/mapbox-gl-js#5063]
Example of the Problem:

![eigzip-problem](https://user-images.githubusercontent.com/25775766/28790433-0818053e-75f7-11e7-9929-8cb6ed8726ac.gif)

**mapbox-gl-js version**: v0.38.0

### Steps to Trigger Behavior

 1. Zoom out past a particular zoom level. The Zipcode layer disappears.

### Expected Behavior

Zoom out to see full United States, and the zip code layers stays intact but diminish the quality of zip code data.

### Actual Behavior

(view GIF above)

Please let me know if you need more information. Thank you in advance!

**Comment by jfirebaugh** [comment]
Hi @gofishwbeeler. Can you please create a minimal self-contained JSFiddle that demonstrates the issue?

**Comment by gofishwbeeler** [comment]
Hi @jfirebaugh, sorry about that. I have a working test here:

https://jsfiddle.net/wbeelergofish/0pmveqwp/

Thanks!
Will

**Comment by jfirebaugh** [comment]
Thanks! It looks like the `gofishwbeeler.2yb69n17` source you're using has a minimum zoom of 5, so Mapbox GL doesn't have any way to render data below that. To learn more about adjusting the zoom extent of your tileset, see [this article](https://www.mapbox.com/help/adjust-tileset-zoom-extent/).

**Comment by gofishwbeeler** [comment]
I apologize.. I ran tippecanoe and created an Mbtiles file. I uploaded it and everything is good to go.. However, when you look at the jsfiddle, you'll see it's not working right (it's not appearing at all).

![screen shot 2017-07-31 at 3 09 21 pm](https://user-images.githubusercontent.com/25775766/28793823-47985c94-7602-11e7-9dff-94d68e578450.png)


https://jsfiddle.net/wbeelergofish/0pmveqwp/1/

Thanks again.

Will
